README.md

```diff
@@ -1,10 +1,10 @@
 # PGo ![CI Status](https://github.com/UBC-NSS/pgo/actions/workflows/ci.yml/badge.svg?branch=master)
```

Comment:
I think maybe it's good to mention MPCal syntax highlighting in the readme.

Reply:
Also a good point, I forgot that.